vmas/simulator/environment/gym.py

```diff
@@ -17,18 +17,25 @@ class GymWrapper(gym.Env):
     def __init__(
         self,
         env: Environment,
+        return_numpy: bool = False,
```

Comment:
```suggestion
```
this wrapper should only return numpy as it returns items and not vectors/tensors